packages/vite-plugin-cloudflare/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @cloudflare/vite-plugin
 
+## 1.0.0
```

Comment:
why is this being marked as a major? I couldn't figure out which changeset marked it so @penalosa @jamesopstad

Reply:
Yeah, this isn't right. It's not obvious to me what's going on here. #8273 changed Wrangler from being a dev dependency to a dependency so maybe that has something to do with it or it could be related to unenv somehow.

Reply:
Maybe a bug in changesets?  https://github.com/changesets/changesets/issues/1011
I suspect it is that we released a major bump to unenv-preset (2.0.0) and vite-plugin has a workspace dependency on that?

Reply:
For now I think we might be able to just manually update this PR before merging...?

Reply:
OK, so looks like this is documented here - https://github.com/changesets/changesets/blob/main/docs/decisions.md#the-versioning-of-peer-dependencies. It seems that if a peer dependency has a minor version bump then the package with the peer dependency is given a major version bump.

We definitely don't want this so I think we have two options:
- Remove the peer dependency on Wrangler altogether (this is only there to try and enforce that the version used for deployment uses the redirected config)
- Use the version range we had before #8273 for the peer dependency rather than `workspace:*`

Reply:
The latter.